pkg/rancher-desktop/main/extensions/manager.ts

```diff
@@ -168,7 +168,21 @@ class ExtensionManagerImpl implements ExtensionManager {
 
     this.setMainHandler('extensions/vm/http-fetch', async(event, config) => {
       const extensionId = this.getExtensionIdFromEvent(event);
-      const url = new URL(config.url);
+      const extension = this.getExtension(extensionId) as ExtensionImpl;
+      let url: URL;
+
+      if (/^[^:/]*:/.test(config.url)) {
+        // the URL is absolute, use as-is.
```

Comment:
This pattern looks to me like it's matching the protocol or domain of a URL, which I suppose makes it absolute.

Reply:
Yep.